src/export/export_format_spaten.hpp

```diff
@@ -0,0 +1,90 @@
+#ifndef EXPORT_EXPORT_FORMAT_SPATEN_HPP
+#define EXPORT_EXPORT_FORMAT_SPATEN_HPP
+
+#include "export_format.hpp"
+
+#include <osmium/fwd.hpp>
+#include <osmium/geom/wkb.hpp>
+#include <osmium/io/writer_options.hpp>
+
+#include <protozero/pbf_builder.hpp>
+
+#include <string>
+
+enum Geom {
+    gt_node = 1,
+    gt_line = 2,
+    gt_poly = 3
+};
+
+enum GeomSerial {
+    wkb = 0
+};
+
+enum class Body : protozero::pbf_tag_type {
```

Comment:
Sure, separated them out. They weren't enum classes, because I didn't know any better. I had to `static_cast` those in order to be able to encode them into the pbf. Is my approach correct?

Reply:
This is one of those places where C++ is a bit annoying. If you use `enum` and not `enum class` for `Geom` and `GeomSerial` and base it on `uint32_t`, you don't need the `static_cast` any more:

```
enum Geom : uint32_t {
...
};
```

Generally I prefer `enum class` over `enum`, but in this case, we can avoid the ugly `static_cast` this way, so I think that's better.